src/Containers/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,166 @@
+import React, {
+    isValidElement,
+    Children,
+    useState,
+    useLayoutEffect,
+    useRef,
+    MutableRefObject,
+    useEffect,
+    ReactElement,
+    ReactNode,
+    useMemo,
+} from 'react';
+import styled from 'styled-components';
+import { transition,scroll } from '../../Utils/Mixins';
+import DropdownItem from './DropdownItem';
+
+type DropdownComponent<P = {}> = React.NamedExoticComponent<P> & {
+  Item: typeof DropdownItem
+}
+
+export interface IDropdownProps {
```

Comment:
Aren't you supposed to be extending the div props here?

Reply:
Yeah I just added the props